x-pack/test/apm_api_integration/tests/agent_explorer/agent_explorer.spec.ts

```diff
@@ -229,6 +251,28 @@ export default function ApiTest({ getService }: FtrProviderContext) {
           expect(body.items).to.be.empty();
         });
       });
+
+      describe('agent latest version', () => {
+        it('returns a version when agent is listed in the file', async () => {
+          const { status, body } = await callApi();
```

Comment:
Does this test require internet access? Do we already require internet access for running the api tests or is this new?

Reply:
Btw. it might not be a bad thing. This way we continuously test the external service so I'd prefer not to mock it out.

Reply:
We already require an internet connection (to the package registry), if I'm not mistaken.

Reply:
This test is accessing the actual resource.
Is there something I need to check in order to allow internet access?